classes/maps.php

```diff
@@ -212,13 +207,13 @@ function generate_iframe() {
 				<html style='margin-top:0 !important;padding-top:0 !important;'>
 				<head>
 					<?php wp_head(); ?>
-					<style type='text/css'>* {
+					<style type='text/css'>
+						body {
+							background: transparent;
 							margin: 0;
 							padding: 0;
-						}</style>
-					<script
-						src="<?php echo esc_url( SIMPLEMAP_MAPS_JS_API . '?v=3&amp;sensor=false&amp;language=' . $atts['default_language'] . '&amp;region=' . $atts['default_country'] ); ?>"
-						type="text/javascript"></script>	
+						}
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->